### PR TITLE
[TACHYON-1582] Restructure POM: rename integration-tests to tests

### PR DESCRIPTION
**Pull request by apc999**
https://tachyon.atlassian.net/browse/TACHYON-1582
This is the followup PR to achieve what proposed in https://tachyon.atlassian.net/secure/attachment/14000/POMstructurechangeproposal.pdf

### CODE REVIEW

File: tests/pom.xml [hunk 1]
```diff
@@ -5,10 +5,10 @@
     <artifactId>tachyon-parent</artifactId>
     <version>0.9.0-SNAPSHOT</version>
   </parent>
-  <artifactId>tachyon-integration-tests</artifactId>
+  <artifactId>tachyon-tests</artifactId>
   <packaging>jar</packaging>
   <name>Tachyon Integration Tests</name>
-  <description>Integration tests on other Tachyon modules</description>
+  <description>Integration tests on different Tachyon modules</description>
```
Comment (jsimsa) [inline tests/pom.xml]: Can we just say "Integration tests for Tachyon"?

**Comment by jsimsa** [comment]
LGTM

**Comment by calvinjia** [comment]
@apc999 you may have missed some references to `integration-tests` (ie. in`AbstractLocalTachyonCluster`)

File: tests/pom.xml (hunk 1 above)
Comment (apc999) [inline tests/pom.xml]: fixed

**Comment by apc999** [comment]
@calvinjia doc updated

**Comment by calvinjia** [comment]
LGTM

**Pull request merged**